examples/jsm/loaders/GLTFLoader.js

```diff
@@ -2840,31 +2842,79 @@ class GLTFParser {
 
 		}
 
+		const promise = this.loadImageSource( sourceIndex, loader ).then( function( texture ) {
+
+			texture.flipY = false;
+
+			if ( textureDef.name ) texture.name = textureDef.name;
+
+			const samplers = json.samplers || {};
+			const sampler = samplers[ textureDef.sampler ] || {};
+
+			texture.magFilter = WEBGL_FILTERS[ sampler.magFilter ] || LinearFilter;
+			texture.minFilter = WEBGL_FILTERS[ sampler.minFilter ] || LinearMipmapLinearFilter;
+			texture.wrapS = WEBGL_WRAPPINGS[ sampler.wrapS ] || RepeatWrapping;
+			texture.wrapT = WEBGL_WRAPPINGS[ sampler.wrapT ] || RepeatWrapping;
+
+			parser.associations.set( texture, { textures: textureIndex } );
+
+			return texture;
+
+		} ).catch( function () {
+
+			console.error( 'THREE.GLTFLoader: Couldn\'t load texture', sourceURI );
+			return null;
+
+		} );
+
+		this.textureCache[ cacheKey ] = promise;
+
+		return promise;
+
+	}
+
+	loadImageSource( sourceIndex, loader ) {
+
+		const parser = this;
+		const json = this.json;
+		const options = this.options;
```

Comment:
This line can be removed.

Reply:
`options` is used at https://github.com/mrdoob/three.js/blob/e4ced3b1b98b488dd906b12fc6edc503ebf66ca7/examples/jsm/loaders/GLTFLoader.js#L2936

Reply:
Strange, I wonder why DeepScan complains then:

https://deepscan.io/dashboard/#view=project&tid=16600&pid=19901&bid=525701&subview=pull-request&prid=1103062

Reply:
Oh wait, it's at a different location.

Reply:
OK, I should have run `npm run lint-examples`. Updated, thanks.